packages/core/src/Scene.ts

```diff
@@ -22,6 +23,8 @@ export class Scene extends EngineObject {
   /** scene-related shaderdata  */
   readonly shaderData: ShaderData = new ShaderData(ShaderDataGroup.Scene);
 
+  /** The background of the scene, which could be a sky or a color. */
+  background: Sky | Color = new Color(0.25, 0.25, 0.25, 1)
```

Comment:
use backgroundMode design，default can be sky.

Reply:
Design reference
```typescript
class Scene
{
   ...
   readonly background: SceneBackground = new SceneBackground();
   ...
}
```
```typescript
enum BackgroundColorMode
{
    Solid,
    Texture,// todo:you can to do later
    Sky
}
```

```typescript
class SceneBackground
{
    clearColor: boolean=true;
    clearDepth: boolean=true;
    clearStencil: boolean=true;

    colorMode: BackgroundColorMode = BackgroundColorMode.Solid;

    colorSolid: Color = new Color(0.25,0.25,0.25,1.0);    
    colorTexture: Texture2D;// todo:you can to do later
    readonly colorSky: Sky = new Sky();
}
```
NOTE:If `colorTexture` and `colorSky` not invaild, for example: `null`, will fall back to solid. 

Usage:
```typeScript
scene.background.colorMode = BackgroundColorMode.Sky;
const sky = scene.background.colorSky;
sky.mesh = PrimitiveMesh.createCube();
sky.material = new AtmosphericSkyMaterial();
```

```typeScript
scene.background.colorMode = BackgroundColorMode.Texture;
scene.background.colorTexture = texture;
```

```typeScript
scene.background.colorMode = BackgroundColorMode.Solid;
scene.background.colorSolid =  new color(1.0,0.0,0.0);
```

Reply:
+1